lucille-examples/lucille-file-to-file-example/conf/connector-for-csv.conf

```diff
@@ -1,6 +1,8 @@
 # this illustrates how the configuration for a connector can be placed in a separate file and referenced from elsewhere
-class: "com.kmwllc.lucille.connector.JSONConnector"
-name: "json_connector"
-jsonPath: "conf/source.jsonl"
+class: "com.kmwllc.lucille.connector.FileConnector"
+name: "connector_for_csv"
```

Comment:
FWIW I'm not sure this renaming is really necessary
I think we can keep json_connector and csv_connector as the names of the connectors and associated conf files even though they use FileConnector...